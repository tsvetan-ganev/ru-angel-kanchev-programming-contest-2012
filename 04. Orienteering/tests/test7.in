25 10
2507Krasteva w 09:50:39 10:35:20
0909Atanasov m 08:12:09 08:42:32
2110Kolev m 09:31:54 10:13:47
1507Getsova w 10:10:31 10:44:00
1710Nikolova w 09:34:31 10:11:36
2301Dyaksova w 10:33:25 11:09:29
1208Damyanov m 09:47:17 10:40:27
0801Tsonkov m 09:13:41 09:59:36
0508Boteva w 10:06:15 10:52:06
2004Vasilev m 08:37:42 09:25:53
0701Zhelyazkova w 09:47:55 10:36:42
1009Dzhulieva w 08:40:47 09:16:51
2201Cholakova w 10:12:45 10:53:47
1107Aleksandrov m 10:11:35 10:56:32
0409Mihaylov m 09:48:49 10:29:46
1802Zasheva w 08:10:57 08:59:06
0103Velkov m 10:33:53 11:23:00
1405Noev m 10:09:09 10:39:32
0308Zhelyazkov m 08:50:35 09:20:03
2404Kostova w 10:14:53 11:03:58
1604Stoeva w 09:31:55 10:14:16
0604Doseva w 09:08:43 09:54:52
0208Ponev m 08:15:20 08:45:43
1904Nankov m 08:00:24 08:50:29
1308Georgiev m 10:34:35 11:23:04