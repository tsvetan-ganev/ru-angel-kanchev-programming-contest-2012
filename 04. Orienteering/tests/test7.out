03 02 09 14 
15 10 23 
08 09